go/test/endtoend/backup/xtrabackup/xtrabackup_test.go

```diff
@@ -41,9 +41,34 @@ func TestXtrabackWithZstdCompression(t *testing.T) {
 	backup.TestBackup(t, backup.XtraBackup, "tar", 0, cDetails, []string{"TestReplicaBackup"})
 }
 
+func TestXtrabackupWithExternalZstdCompression(t *testing.T) {
+	defer setDefaultCompressionFlag()
+	cDetails := &backup.CompressionDetails{
+		CompressorEngineName:    "external",
+		ExternalCompressorCmd:   "zstd",
+		ExternalCompressorExt:   ".zst",
+		ExternalDecompressorCmd: "zstd -d",
+	}
+
+	backup.TestBackup(t, backup.XtraBackup, "tar", 0, cDetails, []string{"TestReplicaBackup"})
```

Comment:
"TestReplicaBackup" is chosen among other test to save test execution time. You can randomly choose other tests (if you want). This will help to give more coverage.

You can ignore this and keep them as you have done right now.